evaluator/evaluate.go

```diff
@@ -100,48 +102,54 @@ func commitWithObserve(tx *gorm.DB) error {
 }
 
 func evaluateAndStore(tx *gorm.DB, system *models.SystemPlatform, vmaasData vmaas.UpdatesV2Response) error {
-	patched, unpatched, err := processSystemAdvisories(tx, system, vmaasData, system.InventoryID)
+	oldSystemAdvisories, patched, unpatched, err := processSystemAdvisories(tx, system, vmaasData, system.InventoryID)
```

Comment:
I see three names for this value: oldSystemAdvisories (here), advisories (method header), stored (when returning in method). Could we unify it, please?